client/app/vps/dashboard/rebuild/vps-rebuild.controller.js

```diff
@@ -0,0 +1,39 @@
+export default class VpsRebuildController {
+  /* @ngInject */
+  constructor(vpsRebuild) {
+    this.vpsRebuild = vpsRebuild;
+  }
+
+  $onInit() {
+    this.vpsOptions = {
+      doNotSendPassword: false,
+    };
+    this.isLoading = true;
+    return this.getAvailableImages()
```

Comment:
Could be extracted in the `routing` and injected no ?

Reply:
I wrote my comment in the wrong place ^^.
This call is not made in routing to avoid user long waiting before the modal shows up

Reply:
I'm not sure what's best, the modal will show up but the user will get a loader and on the other side he'll get a progress loader